client/pom.xml

```diff
@@ -0,0 +1,81 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>online.projektassistent</groupId>
+    <artifactId>projektassistent-parent</artifactId>
+    <version>0.0.2-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>projektassistent-client</artifactId>
+  <packaging>jar</packaging>
+  <name>Projektassistent :: Client</name>
+  <description>Weit e.V. Projektassistent Client</description>
+
+  <build>
```

Comment:
Hier fehlt die configuration, dass der generierte source bei "clean" gelöscht wird. Werden Elemente der OpenAPI Beschreibung gelöscht bzw. unbenannt, dann bleibt der local workspace dirty. Vgl. frontend build:

```xml
      <plugin>
        <artifactId>maven-clean-plugin</artifactId>
        <configuration>
          <filesets>
            <fileset>
              <directory>projects/projektassistent-api-client/src</directory>
              <includes>
                <include>**/*.*</include>
              </includes>
            </fileset>
          </filesets>
        </configuration>
      </plugin>
```
Ist m.E. auch das Problem von https://github.com/DiPA-Projekt/dipa-hub/issues/228

Zusätzlich ist das Ergebnis-Jar vermutlich leer, da der Ergebnis-Ordner des Frontend-Builds nicht als resource für das Jar deklariert ist. Siehe Frontend Build:

```xml
    <resources>
      <resource>
        <directory>dist/projektassistent-frontend</directory>
        <filtering>false</filtering>
        <targetPath>META-INF/resources</targetPath>
      </resource>
    </resources>
```

Reply:
@deleonio  Eigentlich wäre in der Server/pom.xml die folgenden dependency auszutauschen:
```xml
        <dependency>
            <groupId>${project.groupId}</groupId>
            <artifactId>projektassistent-frontend</artifactId>
            <version>${project.version}</version>
        </dependency>
```
Allerdings glaube ich die client/pom.xml produziert im Moment ein leeres Jar wg. der o.g. Punkte. Bei mir lokal kann ich nichts machen, da der Web-Build nicht funktioniert. 
![image](https://user-images.githubusercontent.com/56342691/110777409-5f76ec80-8261-11eb-8b45-11e55dd04744.png)
Mutmaßung, da ist gerade etwas Linux spezifisches drin.

Reply:
Danke @beckerjohannes ,

glaube Deine vielen Kommentare helfen uns weiter. Ja `cp`natürlich ;-)

Ich schaue es mir später an.

VG